### PR TITLE
Update aad-starter's CHANGELOG.md. Make it easier to understand.

### PR DESCRIPTION
**Pull request by chenrujun**
Update aad-starter's CHANGELOG.md. Make it easier to understand.

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @yiliuTo , please help to review this PR.

File: sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md [hunk 1]
```diff
@@ -70,7 +70,12 @@
     azure.activedirectory.user-group.value
     azure.activedirectory.user-group.object-id-key
     ```
-- Stop support of Azure Active Directory Endpoints.
+- Not support `AAD v1` anymore, only support `AAD v2`.
```
Comment (joshfree) [inline sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md:73]: Here's a suggestion for a slight rewording:



```
Removed support for older `AAD v1` style endpoints.
    - Support for `AAD v1`, also named `Azure Active Directory`, endpoints in the form https://login.microsoft.online.com/common/oauth2/authorize has been removed.
    - `AAD v2`, also named `Microsoft Identity Platform`, endpoints in the form https://login.microsoftonline.com/common/oauth2/v2.0/authorize continue to be supported.
    - Please see [this documentation] for more information.
```

**Review by joshfree — APPROVED** [review]

**Review by yiliuTo — APPROVED** [review]
LGTM

**Pull request merged**